software/moltres/wiki/input_example.md

```diff
@@ -483,7 +483,7 @@ $$ \underbrace{\rho c_p \frac{\partial T}{\partial t}}_{[1]}
 The `BCs` block is very similar to the `Kernels` block except the
 `boundary = <boundary_names>` parameter must be specified to indicate where the boundary
 conditions should be applied. The mathematical form of the BCs can be found on
-the [BCs wiki page](/software/moltres/wiki/bcs).
+the [BCs wiki page](https://www.mooseframework.org/modules/porous_flow/boundaries.html).
```

Comment:
Careful! Read the text of the page again carefully and you will realize that this link is supposed to go to to the Moltres wiki. Perhaps make this explicitly clear in the hyperlink? Either way, there is no need to change the linked url.

Reply:
Ah, you're right. I will fix that right now, thank you for noticing.